src/test/java/a_lambda/exercise/Lambdas01Exercise.java

```diff
@@ -32,6 +38,12 @@
     public void sortPersonsByAge() {
         final Person[] persons = getPersons();
         // TODO use Arrays.sort and anonymous class
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                return o1.getAge() - o2.getAge();
+            }
+        });
 
```

Comment:
Do not use subtraction in comparator implementation.

Reply:
Fixed 7acc894e0351a0bd8ad7e6685ea0d3342d0abdf4